### PR TITLE
Remove stackTrace output on caught error.

### PR DESCRIPTION
**Pull request by greebie**
**Remove Stack Trace Output on an Expected Error**

* * *

**GitHub issue(s)**:

#124 

# What does this Pull Request do?

Unit tests were producing a "NullPointerError" that was caught and then thrown during the testing period.  

Since the try-catch call is both expected on a ComputeImageSize(null) and returns (0,0) regardless, there is not reason to output the stack trace here.

# How should this be tested?

Compilation does not print out NullPointer error anymore.

# Additional Notes:
N/A

# Interested parties

Tag (@ mention) interested parties.

Thanks in advance for your help with the Archives Unleashed Toolkit!

### CODE REVIEW

**Comment by greebie** [comment]
The error is thrown when ComputeImageSizeTest purposely (and rightfully) calls ComputeImageSize(null) and expects a return of (0,0).  

The error gets caught by ComputeImageSize who in turn prints the error (needlessly imho) to the screen.  My review of similar tests suggests that printing to screen is not necessary in this particular case.

tl;dr the error you are seeing is expected by the test.  It's just being seen by output due to the printout.

**Comment by greebie** [comment]
Thanks.  Worth keeping this Issue in mind.  The only thing I can think of might be some form of crazy IO error.  In this case, the code will return (0,0) and move on, which is the correct path to take.  The only risk is the very rare case that a crazy IO error causes a different output than usual.  But that's going to be a) quite rare and b) negligible for any large sized dataset.

**Comment by greebie** [comment]
Walking to my car, I wonder if it's worth sending an error warning instead of the stack trace.  I'll mull it over.
